test/binaries/test-rsync-2.5.6.c

```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+
+int main() {
+	printf("This program is designed to test the cve-bin-tool checker.");
+	printf("It outputs a few strings normally associated with rsync 2.5.6");
+	printf("They appear below this line.");
+	printf("------------------");
+	printf("decompressor lost sync!");
+
```

Comment:
Normally you'd include a version string below this line too, if you're not using one to detect the library.